lib/t/news.ex

```diff
@@ -14,12 +14,177 @@ defmodule T.News do
           "background" => %{"color" => "#F97EB9"},
           "labels" => [
             %{
-              "size" => [247.3, 44],
-              "value" => dgettext("news", "Мы стали совсем другими"),
-              "center" => [142.33, 317.83]
+              "zoom" => 1.0,
+              "value" => dgettext("news", "Привет! 👋"),
+              "position" => [24.0, 180.0],
+              "corner_radius" => 0,
+              "alignment" => 0
+            },
+            %{
+              "zoom" => 1.0,
+              "value" =>
+                dgettext("news", "Это новый Since -\nпространство для\nинтересных людей."),
+              "position" => [24.0, 248.0],
+              "corner_radius" => 0,
+              "alignment" => 0
+            },
+            %{
+              "zoom" => 1.0,
+              "value" =>
+                dgettext(
+                  "news",
+                  "К новостям 👉"
+                ),
+              "position" => [150.0, 500.0],
+              "corner_radius" => 0,
+              "alignment" => 0
+            }
+          ],
+          "size" => [375, 667]
+        },
+        %{
+          "background" => %{"color" => "#F97EB9"},
+          "labels" => [
+            %{
+              "zoom" => 1.0,
+              "value" => dgettext("news", "Стикеры-контакты 🔥"),
+              "position" => [24.0, 180.0],
+              "corner_radius" => 0,
+              "alignment" => 0
+            },
+            %{
+              "zoom" => 1.0,
+              "value" => dgettext("news", "Добавляй контакты\nв свою Историю."),
```

Comment:
why capitalized Story?

Reply:
kinda putting greater emphasis on the Story, we did in some other places as well

Reply:
it kinda sucks